src/components/containers/SubplotAccordion.js

```diff
@@ -113,25 +114,28 @@ class SubplotAccordion extends Component {
       }
     });
 
-    let pieCounter = 0;
-    let tableCounter = 0;
+    const counter = {
+      pie: 0,
+      table: 0,
+      sunburst: 0,
+    };
+
     data.forEach((d, i) => {
-      if ((d.type === 'pie' && d.values) || d.type === 'table') {
+      if ((d.type === 'pie' && d.values) || d.type === 'table' || d.type === 'sunburst') {
```

Comment:
Do we need this outer condition? Seems like double work. Perhaps it can be removed? Just have to make sure we still check for `d.values` in `pie` case.